webviz_subsurface/plugins/_seismic_misfit.py

```diff
@@ -3405,7 +3520,9 @@ def make_polygon_df(ensemble_set: dict, polygon: str) -> pd.DataFrame:
                             "(default is None)"
                         )
                     else:
-                        df_polygon["name"] = str(Path(poly_file).name)
+                        name = str(Path(poly_file).name)
+                        name = name.replace(".csv", "").replace("_", " ")
+                        df_polygon["name"] = name
```

Comment:
Maybe you can use (needs to be tested):
```suggestion
                        df_polygon["name"] = Path(poly_file).stem.replace("_", " ")
```

Reply:
Nice one, will fix.